rust-runtime/aws-smithy-wasm/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "aws-smithy-wasm"
+version = "0.1.0"
+authors = [
+  "AWS Rust SDK Team <aws-sdk-rust@amazon.com>",
+  "Eduardo Rodrigues <16357187+eduardomourar@users.noreply.github.com>",
+]
+description = "Smithy WebAssembly configuration for smithy-rs."
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/awslabs/smithy-rs"
+
+[target.'cfg(all(target_family = "wasm", target_os = "wasi"))'.dependencies]
```

Comment:
Is there any harm to making this crate have these dependencies for all targets? I think that would fix the CI issue.

Reply:
Split out the `wasi` crate into its own targeted dependencies section in https://github.com/smithy-lang/smithy-rs/pull/3409/commits/c883aad007e0f5929794be8f5370854679a5f213